DaedalicTestAutomationPlugin/Source/DaedalicTestAutomationPlugin/Private/DaeTestAssertBlueprintFunctionLibrary.cpp

```diff
@@ -4,7 +4,6 @@
 #include "DaeTestTriggerBox.h"
 #include <Blueprint/UserWidget.h>
 #include <Components/Image.h>
-#include <Components/PanelWidget.h>
```

Comment:
Resharper marks this include as unused, but it's necessary to compile the plugin.

Reply:
Yes, it is required: AssertWidgetIsVisible calls !IsValid(Widget->GetParent() which is of type UPanelWidget.

Without that include, I get

`1>DaeTestAssertBlueprintFunctionLibrary.cpp(501): error C2664: 'bool IsValid(const UObject *)': cannot convert argument 1 from 'UPanelWidget *' to 'const UObject *'
`

Reply:
Added include again in commit 6a8cf5a16b4056485aed10e338a04762dfa1a60f.